.github/workflows/release.yaml

```
@@ -0,0 +1,25 @@
+name: Release Rust Crate
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest

```

I think let's add an environment so that we can secure the secret to just this job.

```suggestion
    runs-on: ubuntu-latest
    environment: release
```